pkg/fission-cli/cmd/check/check.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/fission/fission/pkg/fission-cli/cmd"
 	flagkey "github.com/fission/fission/pkg/fission-cli/flag/key"
 	"github.com/fission/fission/pkg/healthcheck"
+	"github.com/pkg/errors"
```

Comment:
Please avoid using pkg/errors. Use `errors` or `fmt.Wrapf` instead.

Reply:
`errors` standard pkg does not have any formatting method.
`fmt.Wrapf` is not available in standard `fmt` pkg so i will use `fmt.Errorf`.
Is this okay?

Reply:
Yes sorry meant Errorf. Basically consider using standard go packages.

Reply:
Done